pkg/kubernetes/kubernetes.go

```diff
@@ -362,13 +362,17 @@ func createCustomResourceDefinition(ctx context.Context, newCrd *apiextensionsv1
 
 	crdName := newCrd.ObjectMeta.Name
```

Comment:
As @BaluDontu  suggested here - https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/744#issuecomment-811489828
@Sophichia  Can you use `crdName` instead of `crd.Name`

```
err = waitForCustomResourceToBeEstablished(ctx, apiextensionsClientSet, crd.Name)
	if err != nil {
		log.Errorf("CRD %q created but failed to establish. Err: %+v", crd.Name, err)
}
```